pkg/server/server.go

```diff
@@ -119,6 +119,7 @@ func (s *Server) init() error {
 	}
 
 	login.Init()
+	login.ProvideService(s.HTTPServer.SQLStore, s.HTTPServer.Login) // XXX hack
```

Comment:
Keep or replace with fake?

Reply:
I think that this provider should be inside `background_services.go` file like the rest of services that are not called directly from the injector system.

Reply:
🤦 I thought this was a test.

Do not replace with a fake 😅